h/views.py

```diff
@@ -113,6 +116,27 @@ def stream(context, request):
         return context
 
 
+@view_config(layout='app', route_name='atom_stream')
+def atom_stream(context, request):
```

Comment:
You don't need the context param if you don't use it. `def atom_stream(request):` will work fine.